### PR TITLE
fix(RatingsBar): fixes the focus on the wrapper

### PR DESCRIPTION
**Pull request by bndiep**
### Overview

Fixes the focus on the RatingsBar wrapper

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [REACH-2203]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

1. Visit http://localhost:6006/?path=/docs/brand-labs-atoms-ratingsbar--ratings-bar
2. Turn VoiceOver on. Navigate to the ratings bar component.
3. Confirm the focus outline surrounds the entire ratings bar and VO announces the %, ratings bar, and level indicator (which is the meter role).

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/708)  | [Portal Env](https://tengu.codecademy.com/learn/learn-python-3?PR_ENV=portal-app-pr-708)   |

[REACH-2203]: https://codecademy.atlassian.net/browse/REACH-2203?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.241-alpha.697bad.0<br/>@codecademy/gamut-labs@33.8.10-alpha.697bad.0<br/>@codecademy/styleguide@62.5.10-alpha.697bad.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6387b625725dda4e0f380a58--gamut-preview.netlify.app>https://6387b625725dda4e0f380a58--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6387b625725dda4e0f380a58>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.241-alpha.e26553.0<br/>@codecademy/gamut-labs@33.8.10-alpha.e26553.0<br/>@codecademy/styleguide@62.5.10-alpha.e26553.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6387dc494a26b97905469da6--gamut-preview.netlify.app>https://6387dc494a26b97905469da6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6387dc494a26b97905469da6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.241-alpha.56b3f9.0<br/>@codecademy/gamut-labs@33.8.10-alpha.56b3f9.0<br/>@codecademy/styleguide@62.5.10-alpha.56b3f9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6387ea6e1d4acf05fc597faa--gamut-preview.netlify.app>https://6387ea6e1d4acf05fc597faa--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6387ea6e1d4acf05fc597faa>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.241-alpha.2734cf.0<br/>@codecademy/gamut-labs@33.8.10-alpha.2734cf.0<br/>@codecademy/styleguide@62.5.10-alpha.2734cf.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6388cbd930b519006c3da94f--gamut-preview.netlify.app>https://6388cbd930b519006c3da94f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6388cbd930b519006c3da94f>Deploy Logs</a></p>

File: packages/gamut-labs/src/RatingsBar/index.tsx [hunk 1]
```diff
@@ -38,8 +37,15 @@ export const RatingsBar: React.FC<RatingsBarProps> = ({
...
 }) => {
   return (
-    <RatingsBarWrapper aria-live="polite">
-      <Text as="label" screenreader>{`${percent}% of ratings`}</Text>
+    <RatingsBarWrapper
+      role="meter"
+      aria-valuenow={percent}
+      aria-valuemin={minimumPercent}
+      aria-valuemax={100}
+      aria-valuetext={`${percent}% of ratings`}
```
Comment (awgraves) [inline packages/gamut-labs/src/RatingsBar/index.tsx:45]: should we also make the announce simply `X percent` and drop the `of ratings` for the same reason of test.io not likely additional words?

Comment (bndiep) [reply packages/gamut-labs/src/RatingsBar/index.tsx:45]: yep! can do.

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6388e8a724ec9c1e1d279573--gamut-preview.netlify.app>https://6388e8a724ec9c1e1d279573--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6388e8a724ec9c1e1d279573>Deploy Logs</a></p>

File: packages/gamut-labs/src/RatingsBar/index.tsx [hunk 2]
```diff
@@ -38,8 +37,14 @@ export const RatingsBar: React.FC<RatingsBarProps> = ({
   percent,
 }) => {
   return (
-    <RatingsBarWrapper aria-live="polite">
-      <Text as="label" screenreader>{`${percent}% of ratings`}</Text>
+    <RatingsBarWrapper
+      role="meter"
+      aria-valuenow={percent}
+      aria-valuemin={0}
+      aria-valuemax={100}
```
Comment (awgraves) [inline packages/gamut-labs/src/RatingsBar/index.tsx:44]: neat!

**Review by awgraves — APPROVED** [review]
🚢

**Pull request merged**